plugins/discourse-details/spec/components/pretty_text_spec.rb

```diff
@@ -6,43 +6,54 @@
   let(:post) { Fabricate(:post) }
 
   it "supports details tag" do
-    cooked_html = <<~HTML.gsub("\n", "")
+    cooked_html = PrettyText.cook <<~MARKDOWN
```

Comment:
None of the tests changed, they were all refactored to use better variable names and the `match_html` matcher.